math/ForceModel.cpp

```diff
@@ -21,7 +21,7 @@
  * along with JuPedSim. If not, see <http://www.gnu.org/licenses/>.
  *
  * \section Description
- * Implementation of the GCFM and the Gompertz model.
+ * Implementation of the GCFM.
```

Comment:
Is this class needed? It is only used in ODESolver, which itself is not used anywhere (as far as I can see)

Reply:
I think you are right. I am cleaning up the models step by step. Will be removed in a following pr.